spec/unit/unparser/emitter/class_methods/handle_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe Unparser::Emitter, '.handle' do
+describe Unparser::Emitter, '.handle', mutant_expression: 'Mutant::Emitter*' do
```

Comment:
No, actulally it should have said `Unparser::Emitter*` but since `--since` is active is was not caught. The fallacies of incremental mutation testing.